src/view/pages/FarmerSearch/index.tsx

```diff
@@ -24,15 +24,17 @@ const defaultFarmerList = [{ name: 'Swalleh', phoneNumber: '1-250-234-1234', id:
                     { name: 'Simon', phoneNumber: '1-234-456-7890', id: 11 }];
 
 /** Basic model for the FarmerType object */
+// TODO: Move to store --@jinglis
 interface FarmerType {
   name: string;
   phoneNumber: string;
   id: number;
 }
 
 /** FarmerSearch OwnPropsType */
+// TODO: Make required property when moving to StorePropsType
 export interface OwnPropsType {
-  listItems?: FarmerType[];
+  farmerSearchListItems?: FarmerType[];
```

Comment:
I think just `farmerList` is most descriptive, because the search items will be a subset of this